docs/content/en/http-methods.md

```diff
@@ -5,54 +5,272 @@ position: 9
 category: API
 ---
 
-<alert type="tip">
+<alert type="info">
 
-See [here](/guide/usage.html#making-requests) for usage information for below methods.
+See [here](/usage#making-requests) for usage information for below methods.
 
 </alert>
 
-<alert type="tip">
+<alert type="info">
 
 Each http method returns a `Promise`
 
 </alert>
 
-### `delete`
-### `get`
+### `$head`
+- arguments: `(url, options?)`
+  - url: `String`
+  - options: [options](/options)
+- resolves: [JSON](https://developer.mozilla.org/en-US/docs/Web/API/Body/json)
+- rejects: `Error`
+
+**Example: **
+```js
+const jsonResponse = await $http.$head('https://unpkg.com/nuxt/package.json')
```

Comment:
Does this return the headers directly?

If so, I would show `const headers = await $http...` with maybe some contents to display the results